pkg/api/types.go

```diff
@@ -580,6 +580,11 @@ type EmptyDirVolumeSource struct {
 	// The default is "" which means to use the node's default medium.
 	// +optional
 	Medium StorageMedium
+	// Total amount of local storage required for this directory.
+	// The default is nil which means that the directory can use all available local storage on the node.
```

Comment:
its not always all available local storage on the node.  maybe just say its undefined if omitted.